### PR TITLE
[Not Entirely Modular] Removes bundles from opfor : Removes Microbomb implants from Ninja + Contractor

### PR DESCRIPTION
**Pull request by RatFromTheJungle**
-
-
## About The Pull Request

following a staff meeting, and per host request, removes the surplus bundles from the opfor panel(as they aren't really approvable), and the microbomb implant from both ninja and contractor

## How This Contributes To The Skyrat Roleplay Experience

Antags opting to immediately qdel themselves when they're about to lose, tends to encourage them to only play when winning

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  
</details>

## Changelog

:cl:
del: Removed Surplus bundles from opfor.
del: Remove microbomb implants from Contractors and Ninjas.
/:cl:

### CODE REVIEW

File: code/modules/antagonists/ninja/outfit.dm [hunk 1]
```diff
@@ -10,7 +10,7 @@
 	internals_slot = ITEM_SLOT_RPOCKET
 	belt = /obj/item/energy_katana
 	back = /obj/item/mod/control/pre_equipped/ninja
-	implants = list(/obj/item/implant/explosive)
+	//implants = list(/obj/item/implant/explosive) SKYRAT EDIT - Removed
```
Comment (Zonespace27) [inline code/modules/antagonists/ninja/outfit.dm]:
```suggestion
	//implants = list(/obj/item/implant/explosive) SKYRAT EDIT REMOVAL
```

**Review by Zonespace27 — CHANGES_REQUESTED** [review]
Give the DC the syndicate weapon implant and implant-lock the DC's contractor baton
I'm concerned about the ninja katana to a lesser degree, so I won't ask for that to be changed for now

**Comment by ghost** [comment]
I understand the bundles plenty, but these two antags have an express goal in mind: don't get taken alive. Their gear is extremely valuable and the station should not ever have it.
Removing their implant that allows them to follow that goal might make them even less likely to fight at all, and we've already got issues with ninjas teleporting too much and contractors taking really gamer loadouts.

Perhaps it could be replaced with an Acidifier instead a'la Rimworld, so rather than exploding their gear is melted?

**Comment by RatFromTheJungle** [comment]
> I understand the bundles plenty, but these two antags have an express goal in mind: don't get taken alive. Their gear is extremely valuable and the station should not ever have it. Removing their implant that allows them to follow that goal might make them even less likely to fight at all, and we've already got issues with ninjas teleporting too much and contractors taking really gamer loadouts.
> 
> Perhaps it could be replaced with an Acidifier instead a'la Rimworld, so rather than exploding their gear is melted?

The current issue; is that because of the lack of 'risk' to losing, that is, in most cases, they just go back to being a ghost: they have no reason to 'rp out a loss,' no reason to even second guess what they bring, and most pertinently, it actively encourages them to just suicide rush if they might lose, which is a real tripple-whammy for an RP level, and it's flip flopping between this, or remove-until-rework

**Review by tf-4 — CHANGES_REQUESTED** [review]
While I understand that the point of this PR is to discourage playing only when winning, which is fine, the bomb implants are there to prevent their stuff being looted. The katana in particular needs to be locked to ninjas just like Zonespace said with the DC baton.

**Comment by RatFromTheJungle** [comment]
> While I understand that the point of this PR is to discourage playing only when winning, which is fine, the bomb implants are there to prevent their stuff being looted. The katana in particular needs to be locked to ninjas just like Zonespace said with the DC baton.

the ninja sword was never locked to their suit(or no-drop, as compared to the contractor baton), and was easier to get then their goggles most of the time (people almost always got it from them) and if we have to start aggressively locking every item they have, if they can't suicide bomb, they'll probably just be snatched out of rotation instead of having to play an eternal wack-a-mole.

(then sit in the limbo of 'can be readded if: x, y, z is done')

**Comment by ghost** [comment]
> I understand the bundles plenty, but these two antags have an express goal in mind: don't get taken alive. Their gear is extremely valuable and the station should not ever have it. Removing their implant that allows them to follow that goal might make them even less likely to fight at all, and we've already got issues with ninjas teleporting too much and contractors taking really gamer loadouts.
> 
> Perhaps it could be replaced with an Acidifier instead a'la Rimworld, so rather than exploding their gear is melted?

Repeating this because it has not been properly addressed. Antags are already more flighty than fighty, making it so not only do they lose but their stuff is then up for grabs and usable against other people isn't how you solve this issue. If anything, it will push people to be even more afraid of combat to a point that they'll cheese things and avoid interaction period.
And no, "administration can just handle that" isn't a solution. Dumping more stuff on their plate to watch out for should be a last resort.
There **needs** to be something preventing the bomb-implant antags from leaving tons of gear behind. Whether that's removing the explosion part of the bomb implant so that it just gibs them, or something like I already suggested which destroys just their gear on death in an immersion-friendly way, I don't care the method. I _do_ care that this doesnt end up as yet another removal to fix one issue that causes another.

**Comment by burgerenergy** [comment]
Contractors are a midround, players get a pop up asking "hey, wanna?" for ten seconds and then fall into the role. There is no real planning, no backstory, no deep lore. It's a chaos agent that we unfortunately rely on too much, but it is what it is. Anything a contractor says or their motives are invented on the spot, which is fine. But because of that I believe it's best to have that unquestionable out, a hard end to the role, mostly because when a contractor fails (and the usually do) then they're stuck with the bill of being in perma and awkwardly trying to figure something out, while forcing security to deal with the milquetoast little midround afterwards - and any prisoners, too, depending on the vibe of the day.

I don't see them being able to qdel themselves "when they're losing" as a bad thing at all. They are on the same cadence of a blob or spicy mold being added to the game, something with a definite end once resolved. If a player wants to keep on trucking its their choice to not use it (and not die, ofc.) The issue is that contractors are currently the leading roleplay-interactive antagonist, which... this will not fix.

also contractor uses your character details and if people just routinely get captured we'll have to ask awkward questions of why Johnny Sims who was captain last round keeps getting thrown into perma as a EOTS contractor and while some people will handwave an excuse some people will just play it straight and i don't want to humor those people, rat

**Comment by RatFromTheJungle** [comment]
> > I understand the bundles plenty, but these two antags have an express goal in mind: don't get taken alive. Their gear is extremely valuable and the station should not ever have it. Removing their implant that allows them to follow that goal might make them even less likely to fight at all, and we've already got issues with ninjas teleporting too much and contractors taking really gamer loadouts.
> > Perhaps it could be replaced with an Acidifier instead a'la Rimworld, so rather than exploding their gear is melted?
> 
> Repeating this because it has not been properly addressed. Antags are already more flighty than fighty, making it so not only do they lose but their stuff is then up for grabs and usable against other people isn't how you solve this issue. If anything, it will push people to be even more afraid of combat to a point that they'll cheese things and avoid interaction period. And no, "administration can just handle that" isn't a solution. Dumping more stuff on their plate to watch out for should be a last resort. There **needs** to be something preventing the bomb-implant antags from leaving tons of gear behind. Whether that's removing the explosion part of the bomb implant so that it just gibs them, or something like I already suggested which destroys just their gear on death in an immersion-friendly way, I don't care the method. I _do_ care that this doesnt end up as yet another removal to fix one issue that causes another.


I didn't ask for this to be reopened myself, but, if the items are so unbelievably overpowered that no one else should /ever/ get their hands on it; they shouldn't have the items to begin with, and people can adjust accordingly if this is merged

**Comment by RatFromTheJungle** [comment]
> Contractors are a midround, players get a pop up asking "hey, wanna?" for ten seconds and then fall into the role. There is no real planning, no backstory, no deep lore. It's a chaos agent that we unfortunately rely on too much, but it is what it is. Anything a contractor says or their motives are invented on the spot, which is fine. But because of that I believe it's best to have that unquestionable out, a hard end to the role, mostly because when a contractor fails (and the usually do) then they're stuck with the bill of being in perma and awkwardly trying to figure something out, while forcing security to deal with the milquetoast little midround afterwards - and any prisoners, too, depending on the vibe of the day.
> 
> I don't see them being able to qdel themselves "when they're losing" as a bad thing at all. They are on the same cadence of a blob or spicy mold being added to the game, something with a definite end once resolved. If a player wants to keep on trucking its their choice to not use it (and not die, ofc.) The issue is that contractors are currently the leading roleplay-interactive antagonist, which... this will not fix.
> 
> also contractor uses your character details and if people just routinely get captured we'll have to ask awkward questions of why Johnny Sims who was captain last round keeps getting thrown into perma as a EOTS contractor and while some people will handwave an excuse some people will just play it straight and i don't want to humor those people, rat

i'll respond with, to everything, 'we're trying to be a roleplay server firstly, and foremostly, if contractors can't roleplay after losing, they should just be disabled aswell' and that'll be my total ruling on it, if they aren't contributing to roleplay, and only play 'when they're winning,' they're not a viable long-term antag, and should be taken out of rotation for not contributing to the servers state in a healthy manner, it's hardly roleplay if you drop it to RR yourself the moment you're no longer running in circles.

**Review by GoldenAlpharex — APPROVED** [review]
I'm willing to give this a shot, and it's not as if it was hard to revert later either, it's like two clicks.

**Comment by tf-4** [comment]
isnt that the point of a test merge

**Pull request merged**